src/main/java/io/moderne/devcenter/DependencyVulnerabilityCheck.java

```diff
@@ -130,18 +144,15 @@ public List<DevCenterMeasure> getMeasures() {
     @RequiredArgsConstructor
     @Getter
     public enum UpgradeDeltaMeasure implements DevCenterMeasure {
-        Major("Major version bump required."),
-        Minor("Minor version bump required."),
-        Patch("Patch version bump required."),
-        Completed("No fixable vulnerabilities found.");
+        Major("Major", "Some dependencies require a major version bump."),
+        Minor("Minor", "Some dependencies require a minor version bump."),
+        Patch("Patch", "Some dependencies require a patch version bump."),
+        NotFixable("No fix available", "No fix is available for the vulnerabilities that are present"),
+        Completed("No vulnerabilities", "No fixable vulnerabilities found.");
```

Comment:
Should this then be: 
```suggestion
        Completed("No vulnerabilities", "No vulnerabilities found.");
```
?